velox/exec/HashAggregation.cpp

```diff
@@ -385,6 +399,11 @@ RowVectorPtr HashAggregation::getDistinctOutput() {
 }
 
 void HashAggregation::noMoreInput() {
+  auto mergeVector = mergingVectorInput_->getVector(true);
```

Comment:
Is it possible that we can get more than one vector from `meringVectorInput_`, just like what we did in `addInput`?

Reply:
There will not be multiple merged vectors here, because every time call getVector() in addInput() will take away all the merged vectors, so there will be at most 1 merged vector when calls getVector() in noMoreInput().  

To make it easier to understand the code, I will extract a common method.